numpy/core/src/umath/loops.c.src

```diff
@@ -896,7 +896,7 @@ NPY_NO_EXPORT void
 NPY_NO_EXPORT void
 @TYPE@_@kind@(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
 {
-    OUTPUT_LOOP_FAST(npy_bool, *out = @val@);
+    UNARY_LOOP_FAST(@type@, npy_bool, *out = @val@);
```

Comment:
Does changing this to `(void) in; *out = @val@` do the trick?

Reply:
Looks promising - no warnings locally and no performance regression. Just rebased and pushed, so will see what Travis has to say

Reply:
Might be worth adding a comment explaining the `(void)` - that the macro provides an `in` variable, and we need to tell the compiler we are deliberately ignoring it

Reply:
Done